exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -106,6 +106,7 @@
   <p>STATE: ${model.getProfile().getState().name()}</p>
   <p>FOREMAN: ${model.getProfile().getForeman().getAddress()}</p>
   <p>TOTAL FRAGMENTS: ${model.getProfile().getTotalFragments()}</p>
+  <p>PLANNING: ${model.getPlanningDuration()}</p>
```

Comment:
Is planning part of duration? Or, is total time planning time + (execution) duration?

If planning time is part of the (overall) duration, may be show that visually somehow. Indented? And, if we show planning time, should we also not show queue time and execution time?